### DISCUSSION TITLE
20241114 Weekly Development Meeting

### DISCUSSION THREAD

**Discussion by josecelano** [nautilus-cyberneering/.github#116] [Meetings]
- Time:  9:30 in Canary Islands (WET (UTC+0)).
- Time: 10:30 in Vaduz (CET (UTC+1)).

Location:
<https://meeting.blockfinance-eco.li/NautilusCyberneeringWeeklyDevMeeting>

Expected duration: 2 to 4 hours with breaks

Communicated attendants:

- Cameron Garnham(@da2ce7)  - x 
- Jose Celano(@josecelano) - x
- Constantin Bosse(@cgbosse) - x
- Mario Melchor(@mario-nt) - x
- Graeme Byrne(@grmbyrn) - x

-------------------------------------------------------------

## Agenda:

1. Everyone will take their turn and:
- Go over his notes added as comments for the meeting which will have been added in advance to give an update of all the work done during the previous week indicating.
	- The issues worked on
	- The progress made on them
	- Other things done
- Inform of any faced challenges and request help if necessary to work on specific issues.
- Highlight the encountered difficulties, especially if there is a risk of them causing delays or affecting end product's quality.
- Communicate what will be the next week's tasks.
- List any new topics for further discussion in an additional Technical meeting that he noted for future discussion in comments to this discussion either before the meeting or after his turn so that they get considered.

During the meeting every team member will take their turns and follow this sequence.
The topics for further discussion will be noted in advance by each the team member and to the meetings agenda.

2. At the end of the meeting if there any new technical topics to be discussed further it will be decided to
set another meeting date for the discussion of these or to hold the meeting hereafter.

Also the persons to be part of the meeting will be decided on so that the others will be free to continue their own work.

**Comment by josecelano** [comment]
### Code Coverage Report

We have to change the way we create and upload the code coverage to [codecov](https://about.codecov.io/).

I have created an example: https://github.com/josecelano-test/pull_request_target

In the example, we are not using the `alekitto/grcov@v0.2`. The action has not been updated to fix this [issue](https://github.com/torrust/torrust-tracker/issues/1075).

### Bencode2Json

Dicussion with @da2ce7: https://github.com/torrust/bencode2json/issues/7

### Community support

- I had a meeting (approx 2h) with two Rust developers to introduce them to Torrust. One of them wants to start contributing.

### Torrust Website

- I've installed the Index+Tracker in a new VM from scratch (3.5h) and [updated the blog article with minor changes](https://github.com/torrust/torrust-website/pull/109).

The process is very tedious. It would be convenient to build at least a simple installer. I think this could be a common way to deploy Torrust.

I've also opened new issues:

- https://github.com/torrust/torrust-website/issues/108
- https://github.com/torrust/torrust-website/issues/110

PR review:

- https://github.com/torrust/torrust-website/pull/111

### Torrust Index GUI

Comment on issues:

- https://github.com/torrust/torrust-index-gui/issues/635
- https://github.com/torrust/torrust-index-gui/issues/637
- https://github.com/torrust/torrust-index-gui/discussions/638

### Torrust Tracker

New PR:

- https://github.com/torrust/torrust-tracker/pull/1081

### Torrust Demo

New issue:

- https://github.com/torrust/torrust-demo/issues/16

**NOTE:** This week, I've worked 14h approx on Torrust.

**Comment by mario-nt** [comment]
### GUI
- https://github.com/torrust/torrust-index-gui/discussions/638
- https://github.com/torrust/torrust-index-gui/issues/637
- https://github.com/torrust/torrust-index-gui/issues/428

### Training and research

- https://flexboxfroggy.com/

### Other

- Research on how to close Snyk automatic PRs

**Comment by grmbyrn** [comment]
Torrust Website:
* [fix search error for no results](https://github.com/torrust/torrust-website/pull/111)
* [add melt-ui toc in (pages)](https://github.com/torrust/torrust-website/pull/113)
* [remove vertical scroll bar](https://github.com/torrust/torrust-website/pull/116)

Study:
* [Scrimba TypeScript Course](https://v2.scrimba.com/learn-typescript-c03c)
